Source/WebCore/platform/SharedBuffer.h

```diff
@@ -26,6 +26,8 @@
 
 #pragma once
 
+#include "SharedMemory.h"
```

Comment:
Are you really just we can't just do `class SharedMemoryHandle;` to forward-declare? :)

Reply:
I'll try it again